src/Microsoft.TestPlatform.CrossPlatEngine/Client/ProxyOperationManager.cs

```diff
@@ -224,7 +228,9 @@ public virtual void Close()
         /// </returns>
         protected virtual TestProcessStartInfo UpdateTestProcessStartInfo(TestProcessStartInfo testProcessStartInfo)
         {
-            // do nothing. 
+            // Update Telemetry Opt in status because by default in Test Host Telemetry is opted out
+            var telemetryOptedIn = this.requestData.IsTelemetryOptedOut ? "false" : "true";
```

Comment:
>IsTelemetryOptedOut [](start = 52, length = 19)

Better use either OptIn or OptOut everywhere.
Or simply call it EnableTelemetry, maybe?

Reply:
done